msal/src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -95,7 +95,33 @@ public PublicClientApplication(string clientId, string authority)
             {
                 ClientId = clientId
             };
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="scopes"></param>
+        /// <param name="extraQueryParameters"></param>
+        /// <param name="deviceCodeResultCallback"></param>
+        /// <returns></returns>
+        public async Task<AuthenticationResult> AcquireTokenAsync(

```

Comment:
>public async Task<AuthenticationResult> AcquireTokenAsync( [](start = 8, length = 58)

naming, AquireTokenWithDeviceCodeAsync .... and per other comment, this might have to move to specific platforms.